todocore/src/test/java/com/knighttodo/todocore/integration/DayTodoResourceIntegrationTest.java

```diff
@@ -329,4 +331,17 @@ public void findDayTodosByDayId_shouldReturnExistingTodo_whenIdIsCorrect() throw
                 .andExpect(jsonPath(buildJsonPathToLength()).value(2));
     }
 
+    @Test
+    public void addDayTodo_shouldRespondWithBadRequestStatus_whenColorIsNull() throws Exception {
+        Day day = dayRepository.save(DayFactory.dayInstance());
+        DayTodoRequestDto requestDto = DayTodoFactory.createDayTodoRequestDtoWithoutColor();
+
+        mockMvc.perform(post(API_BASE_URL_V1 + API_BASE_DAYS + "/" + day.getId() + API_BASE_TODOS)
+                        .content(objectMapper.writeValueAsString(requestDto))
+                        .contentType(MediaType.APPLICATION_JSON_VALUE))
+                .andExpect(status().isBadRequest());
+
+        assertThat(dayTodoRepository.count()).isEqualTo(0);
+    }
```

Comment:
I suppose that this test won't be needed. We should discuss such stuff that related to business logic separately so we don't spend time on redundant work